snapcraft/commands/microcloud

```diff
@@ -14,6 +14,15 @@ fi
 export VIMRUNTIME="/snap/microcloud/current/usr/share/vim/vim91"
 export EDITOR="vim.tiny -Z --clean"
 
+# Set feature flag environment variable based on the snap configuration
+# This can be set/unset with the following commands:
+# snap set microcloud feature.cluster-manager=true
+# snap unset microcloud feature.cluster-manager
```

Comment:
Setting/unsetting seem to require a `configure` hook script:

```
root@micro1:~# snap set microcloud feature.cluster-manager=true
error: cannot perform the following tasks:
- Run configure hook of "microcloud" snap (snap "microcloud" has no "configure" hook)
```

Such script could be a simpler version than https://github.com/canonical/lxd-pkg-snap/blob/latest-edge/snapcraft/hooks/configure

I think we'd need a simple script in `snap/hooks/configure` that persists the set key to disk and tweak the `microcloud` command to read from it.

I'm happy to deal with that rework if you'd like @edlerd.

Reply:
I added the configure hook during development. But then removed it, and it kept on working for me. Did you test this with a snap build from this PR?